### ISSUE TITLE
Is there a way to build with relative paths?

### ISSUE THREAD

**Issue by tiptronic** [gridsome/gridsome#575]
By default, Gridsome loads all assets from the absolute root of my site.
I can add `pathPrefix` to move that to a subfolder (e.g. `mysubfolder`), but all assets are still loaded from an absolute url: `/mysubfolder/...`.

Is there a way to have Gridsome to load **relative** paths?  (also all of Gridsome's assets and css)?

**Comment by hjvedvik** [comment]
No, unfortunately :) It might be doable with the `<base>` HTML element, but it would require some changes in the webpack config, etc, if it is possible.

**Comment by tiptronic** [comment]
Can I add a feature request for this?
I think this would be a very valuable addition to work out of the box!

**Comment by Spica2** [comment]
+1

**Comment by aaronaat** [comment]
Please add, it's necessary for IPFS deployment!

**Comment by OliPerraul** [comment]
@tiptronic did you find a solution for this I am facing the same issue? Thanks

**Comment by tiptronic** [comment]
Hi @OliPerraul Yes - I used another framework :)

I am really bored, if basic functionality is ignored and they try to sell it as 'feature'...

I like Gridsome pretty much, but I simply don't have to fight for such a basic thing...
-> I'm back, when it works... (if ever)

**Comment by spacedawwwg** [comment]
Balls.

**Comment by kraeki** [comment]
Can we add some prio to this? I am blocked in deployment in this moment. Happy to help but would need some overview.

**Comment by carsonbird** [comment]
+1 How is this not already implemented? Why assume users never ever host from a subfolder?

**Comment by animald** [comment]
We are building out landing pages for our clients that will either be downloaded by them, or deployed to their servers by us. Therefore, it isn't realistic for us to assume the eventual path of the landing page itself. 

PLEASE make it possible to build with relative paths.

In the meantime if anyone has a solution for our problem that doesn't involve pulling away from Gridsome then please let me know.

Thanks!

**Comment by kraeki** [comment]
It would also help us in deploying it with Gitlab Pages...

**Comment by lirumeng** [comment]
Is there any follow-up?
